Assignment8 Exception/src/Main.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Optional;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+
+        /*using Exception      (checked) */
+
+        StudentException studentException=new StudentException("1");
+
+        try {
+            String s=studentException.getStudentId();
+            System.out.println(s);
+        } catch (StudentException e) {
+
+            e.printStackTrace();
+        }
+
+
+
+        /* using Runtime Exception      (unchecked)*/
+        StudentRuntimeException studentRuntimeException =new StudentRuntimeException("1");
```

Comment:
What is this? why you are create instance of exception ..we usually throw exception when needed